proposed/2021/NuGet.orgTFMs.md

```diff
@@ -0,0 +1,96 @@
+# NuGet Target Frameworks on Package Details Page 
+
+- Author Name (https://github.com/jcjiang)
+- Start Date (2021-02-16)
+- GitHub Issue (https://github.com/NuGet/NuGetGallery/issues/8387)
```

Comment:
There are a bajillion issues about package compatibility. Have we scoured them for clues on how to improve this limited-scope proposal? Many asks are about searching or VS UI which are out of scope, but I imagine some feedback can be considered right now.

Reply:
E.g. https://github.com/NuGet/NuGetGallery/issues/3098

Reply:
This issue have many links to and from it.

Reply:
I believe https://github.com/NuGet/NuGetGallery/issues/4843 is the best issue to describe this scope.

Reply:
I love how these 4-year-old conversations just get peeled off in these discussions like they were sitting at top of mind for the encyclopedic amongst us. :-D

Reply:
I've moved @terrajobst's issue front and center in this spec, and will close the tracking issue I've been using and use Immo's.